src/styles/_header.scss

```diff
@@ -0,0 +1,69 @@
+@import "variables";
+
+.logo {
+  margin-left: 50px;
```

Comment:
<img width="1016" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/86789161/c982014c-8f8c-4f77-96b4-7e6788007b66">
Fix it in all places